proposals/image_verify_enhancements.md

```diff
@@ -294,23 +328,29 @@ spec:
       resources:
         kinds:
         - tekton.io/v1/Task
-    context:
-    - imagePaths:
-      - path: "{{ spec.tasks[*] }}"
     imageVerify:
+      imagePaths:
+      - path: "{{ spec.tasks[*] }}"
       default: deny
       digestVerify: true
       images:
       - "registry.company.com/*"
       attestors:
       - match:
         entries:
-        - keyless: {}
+          - keyless:
+            subject: "https://github.com/JimBugwadia/demo-java-tomcat/.github/workflows/publish.yaml@refs/tags/*"
+            issuer: "https://token.actions.githubusercontent.com"
+            additionalExtensions:
+              githubWorkflowTrigger: push
+              githubWorkflowSha: 73f9df28b6c67e3d4a4ffc4b75aaeed89be88b58
+              githubWorkflowName: build-sign-attest
+              githubWorkflowRepository: JimBugwadia/demo-java-tomcat
 ```
 
 #### Example: require all pod images are have a Trivy image scan attestation with keyless signing
```

Comment:
Example has a public key instead of keyless. Or Trivy attestations use public key only?